ams/api/src/main/java/com/netease/arctic/ams/api/metrics/TaggedMetrics.java

```diff
@@ -32,56 +32,56 @@
 import java.util.Map;
 
 /**
- *  This is a simple data structure that separates tags and metrics, making it easier for reporters to write to popular
- *  monitoring systems when processing {@link MetricReport}
+ * This is a simple data structure that separates tags and metrics, making it easier for reporters to write to popular
+ * monitoring systems when processing {@link MetricsContent}
  */
 public class TaggedMetrics {
   private final Map<String, Object> tags;
-  private final Map<String, Object> metrics;
+  private final Map<String, com.codahale.metrics.Metric> metrics;
```

Comment:
Why id the full class name used here?  I did not see a class name conflicts in this class.

Reply:
There is a conflict between the Metric annotation and the Metric class in the TaggedMetrics class.